src/server/scripts/Northrend/Ulduar/Ulduar/boss_kologarn.cpp

```diff
@@ -431,34 +436,14 @@ class boss_kologarn : public CreatureScript
                     return;
                 case EVENT_FOCUSED_EYEBEAM:
                 {
-                    events.ScheduleEvent(EVENT_FOCUSED_EYEBEAM, 13000+rand()%5000);
-                    Unit* target = nullptr;
-                    Map::PlayerList const& pList = me->GetMap()->GetPlayers();
-                    for(auto itr = pList.begin(); itr != pList.end(); ++itr)
-                    {
-                        if (itr->GetSource()->GetPositionZ() < 420)
-                            continue;
+                    events.ScheduleEvent(EVENT_FOCUSED_EYEBEAM, urand(15000,25000));
```

Comment:
```cpp
events.ScheduleEvent(EVENT_FOCUSED_EYEBEAM, 10000);
```

Reply:
Hi, it should be 20000 not 10000, you have to take in account the beam duration as well

Reply:
@wizzymore  oops ! , right.